fulp_modules/features/antagonists/bloodsuckers/code/clans/clan_brujah.dm

```diff
@@ -0,0 +1,102 @@
+/datum/bloodsucker_clan/brujah
+	name = CLAN_BRUJAH
+	description = "The Brujah seek societal advancement through direct (and usually violent) means.\n\
+		With age they develop a powerful physique and become capable of obliterating almost anything with their bare hands.\n\
+		Be wary, as they are ferverous insurgents, rebels, and anarchists who always attempt to undermine local authorities. \n\
+		Their favorite vassal gains the regular Brawn ability and substantially strengthened fists."
+	clan_objective = /datum/objective/brujah_clan_objective
+	join_icon_state = "brujah"
+	join_description = "Gain an enhanced version of the brawn ability that lets you destroy most structures (including walls!) \
+		Rebel against all authority and attempt to subvert it, but in turn <b>break the Masquerade immediately on joining</b>  \
+		and lose nearly all of your Humanity."
+	blood_drink_type = BLOODSUCKER_DRINK_INHUMANELY
+
+
+/datum/bloodsucker_clan/brujah/New(datum/antagonist/bloodsucker/owner_datum)
+	. = ..()
+	owner_datum.special_vassals.Cut() //Removes Discordant Vassal, which is in the list by default.
+	owner_datum.break_masquerade()
+	owner_datum.AddHumanityLost(37.5) // Frenzy at 400
+	bloodsuckerdatum.remove_nondefault_powers(return_levels = TRUE)
+	owner_datum.powers += new /datum/action/cooldown/bloodsucker/targeted/brawn //Doesn't actually give brawn, but makes it unpurchasable.
```

Comment:
`BRUJAH_CAN_BUY` just makes the power be innately gained upon joining the Brujah clan as seen in the four lines immediately following 21 here. To make it prevent the purchase of Brawn I'd need to add it to every other regular Bloodsucker ability and rework current Brujah power purchasing code. The current solution is simpler and functional, but if you'd like then I could try something else.

Reply:
I'm thinking using abstract type which you can use to block getting 2 powers of the same type. I think heretic might use that if needed

Reply:
I think I halfway understand what you mean... Where might I find a good reference for that in heretic code?

Reply:
You can see ``try_add_objective`` on traitor which prevents them from getting duplicate objectives, ``abstract_parent_type`` for heretics

Reply:
Thank you for the references— they do help a lot.

Still, I just looked at the system and it seems a bit too complex for me to both completely understand and implement. If I were to add it in here then I'd _only_ be doing so for the powers Brawn and Brash (and I probably wouldn't be able to create a functional groundwork for the later expansion of the system either). Anything beyond that seems like it would be a larger code change warranting its own PR. 

If I've misunderstood the system and implementing it would be much easier (i.e., not involving a substantial rework of power selection code) than I expect then I do apologize.

Reply:
Do we think this review is resolved? If so we can get this TMed